src/components/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import '../styling/App.css';
+import { Route, Switch } from 'react-router';
+import { Provider } from 'react-redux';
+import Books from './Books';
+import Navbar from './Navbar';
+import Categories from './Categories';
+import store from '../store';
+
+function App() {
+  return (
+    <Provider store={store}>
+      <Navbar />
+      <Switch>
+        <Route path="/books">
+          <Books />
+        </Route>
```

Comment:
- [ ] Kindly let's have the `Books` component as the default route for now, as mentioned in the project requirement because for now, it's the `Categories` component that seems to be the default route. The default route should looks like this 👉  `<Route path="/"><Books /></Route>`, don't delete the current route you have, you only need to add this new one above ☝️